packages/bot-web-ui/src/pages/bot-builder/quick-strategy/types.ts

```diff
@@ -129,3 +129,26 @@ export type TFormValues = Record<string, string | number | boolean> & {
     type?: string;
     duration?: string;
 };
+
+export type TContract = {
+    transaction_ids: {
+        buy?: string;
+        sell?: string;
+    };
+    tick_count?: number;
+    barrier?: string;
+    high_barrier?: string;
+    low_barrier?: string;
+    date_start?: string;
+    entry_tick?: number;
+    entry_tick_time?: string;
+    exit_tick?: number;
+    exit_tick_time?: string;
+    underlying: string;
+    display_name: string;
+    contract_type: string;
+    buy_price: number;
+    currency: string;
+    is_completed: boolean;
+    profit: number;
+};
```

Comment:
@farabi-deriv can we use `ProposalOpenContract` from @deriv/api-types package?

Reply:
@sandeep-deriv sure we can use it from deriv/api-types.